packages/perseus-editor/src/__stories__/editor.stories.tsx

```diff
@@ -1,17 +1,21 @@
-import {ApiOptions} from "@khanacademy/perseus";
+/* eslint-disable react/prop-types */
+import {ApiOptions, PerseusRenderer} from "@khanacademy/perseus";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {action} from "@storybook/addon-actions";
 import * as React from "react";
 
+import SideBySide from "../../../../testing/side-by-side";
 import {question1} from "../__testdata__/input-number.testdata";
 import Editor from "../editor";
 import {registerAllWidgetsAndEditorsForTesting} from "../util/register-all-widgets-and-editors-for-testing";
 
+registerAllWidgetsAndEditorsForTesting();
```

Comment:
a shame we have this top-level side-effect. Is there some "storybook setup" file this could go in?

Reply:
Good call.  Moving this to `storybook/preview.js` which is the global setup file!